src/s3/commands/presignedURL.ts

```diff
@@ -38,36 +45,124 @@ export async function presignedURLCommand(
         })
 
         await copyToClipboard(url, 'URL', window, env)
+        vscode.window.showInformationMessage(`A presigned URL has been copied to your clipboard`)
```

Comment:
If we are concerned that the statusbar message isn't visible enough, let's address that in `copyToClipboard()` rather than special-case the Preseigned URL feature. And then we would probably want a "Don't show again" button if we go in that direction.

To keep this PR in scope, consider creating a backlog item for that and drop this info message for now.

Reply:
Backlog item created, notification removed.